Gemfile

```diff
@@ -47,5 +47,5 @@ gem "pundit", "~> 2.1"
 gem "redcarpet", "~> 3.5"
 gem "sidekiq", "~> 6.3"
 gem "stripe", ">= 2.8", "< 6.0"
-
+gem "sitemap_generator"
```

Comment:
Please move this one line above and sort the chunk of gems.

Reply:
Updated.
 I thought these names are started with **s**. 🤣